src/server/server.py

```diff
@@ -190,6 +201,9 @@ async def start_server(self, on_connect: Callable = None):
         self.log.info(f"Started listening on port: {self._port}")
 
     async def incoming_connection(self, request):
+        if request.remote in self.banned_peers and time.time() < self.banned_peers[request.remote]:
```

Comment:
I see the other banned periods but where are you setting the general ban period?

Reply:
This is a work in progress. The constructor for ChiaServer has some ban times.